backend/src/test/java/com/onwelo/practice/bts/service/BankAccountServiceTest.java

```diff
@@ -1,24 +1,141 @@
 package com.onwelo.practice.bts.service;
 
 import com.onwelo.practice.bts.entity.BankAccount;
+import com.onwelo.practice.bts.entity.Transfer;
+import junit.framework.Assert;
 import org.junit.Test;
 import org.junit.runner.RunWith;
 import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
 import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
-import org.springframework.util.Assert;
+
+import java.util.ArrayList;
+import java.util.List;
 
 @RunWith(SpringJUnit4ClassRunner.class)
+@SpringBootTest
 public class BankAccountServiceTest {
+
     @Autowired
     private BankAccountService bankAccountService;
 
+    @Autowired
+    private TransferService transferService;
+
+    @Test
```

Comment:
Mówiliśmy coś o najnowszej wersji JUnit (tj. 5). Zmieniłbym to na nią do puki nie ma jeszcze zbyt wiele kodu. Jest tam sporo fajnych opcji które upraszczają pisanie testów.